src/status_im/contexts/wallet/common/asset_list/view.cljs

```diff
@@ -1,36 +1,44 @@
 (ns status-im.contexts.wallet.common.asset-list.view
   (:require
     [quo.core :as quo]
-    [react-native.core :as rn]
+    [react-native.gesture :as gesture]
     [status-im.contexts.wallet.common.utils :as utils]
     [utils.re-frame :as rf]))
 
 (defn- asset-component
-  [token _ _ {:keys [currency currency-symbol on-token-press]}]
+  [{token-symbol :symbol
```

Comment:
Thanks for taking care of not shadowing Clojure vars!

Reply:
yes!, we've had many issues because of it